funcx_endpoint/funcx_endpoint/endpoint/interchange.py

```diff
@@ -629,265 +431,4 @@ def _main_loop(self):
-    # comm_q.put((ic.worker_task_port,
-    #            ic.worker_result_port))
-    ic.start()
-
-
-def cli_run():
-
-    parser = argparse.ArgumentParser()
-    parser.add_argument("-c", "--client_address", required=True, help="Client address")
-    parser.add_argument(
-        "--client_ports",
-        required=True,
-        help="client ports as a triple of outgoing,incoming,command",
-    )
-    parser.add_argument("--worker_port_range", help="Worker port range as a tuple")
-    parser.add_argument(
-        "-l",
-        "--logdir",
-        default="./parsl_worker_logs",
-        help="Parsl worker log directory",
-    )
-    parser.add_argument(
-        "--worker_ports",
-        default=None,
-        help="OPTIONAL, pair of workers ports to listen on, "
-        "e.g. --worker_ports=50001,50005",
-    )
-    parser.add_argument(
-        "--suppress_failure",
-        action="store_true",
-        help="Enables suppression of failures",
-    )
-    parser.add_argument(
-        "--endpoint_id",
-        required=True,
-        help="Endpoint ID, used to identify the endpoint to the remote broker",
-    )
-    parser.add_argument("--hb_threshold", help="Heartbeat threshold in seconds")
-    parser.add_argument(
-        "--config",
-        default=None,
-        help="Configuration object that describes provisioning",
-    )
-    parser.add_argument(
-        "-d", "--debug", action="store_true", help="Enables debug logging"
-    )
-
-    print("Starting HTEX Intechange")
-    args = parser.parse_args()
-
-    setup_logging(logfile=os.path.join(args.logdir, "endpoint.log"), debug=args.debug)
-
-    optionals = {}
-    optionals["suppress_failure"] = args.suppress_failure
-    optionals["logdir"] = os.path.abspath(args.logdir)
-    optionals["client_address"] = args.client_address
-    optionals["client_ports"] = [int(i) for i in args.client_ports.split(",")]
-    optionals["endpoint_id"] = args.endpoint_id
-
-    # DEBUG ONLY : TODO: FIX
-    if args.config is None:
-        from parsl.providers import LocalProvider
-
-        from funcx_endpoint.endpoint.utils.config import Config
-
-        config = Config(
-            worker_debug=True,
-            scaling_enabled=True,
-            provider=LocalProvider(
-                init_blocks=1,
-                min_blocks=1,
-                max_blocks=1,
-            ),
-            max_workers_per_node=2,
-            funcx_service_address="http://127.0.0.1:8080",
-        )
-        optionals["config"] = config
-    else:
-        optionals["config"] = args.config
-
-    if args.worker_ports:
-        optionals["worker_ports"] = [int(i) for i in args.worker_ports.split(",")]
-    if args.worker_port_range:
-        optionals["worker_port_range"] = [
-            int(i) for i in args.worker_port_range.split(",")
-        ]
-
-    ic = EndpointInterchange(**optionals)
-    ic.start()
-
-    """
-    with daemon.DaemonContext():
-    """
+        self._quiesce_complete.set()
```

Comment:
Not a comment on the change itself, more of a meta-comment.
Oy vey, GitHub! Placing this after the negative diff is just... 🙄